src/main/java/org/junitpioneer/jupiter/RetryingTestExtension.java

```diff
@@ -82,7 +84,37 @@ static FailedTestRetrier createFor(Method test) {
 			RetryingTest retryingTest = AnnotationSupport
 					.findAnnotation(test, RetryingTest.class)
 					.orElseThrow(() -> new IllegalStateException("@RetryingTest is missing."));
-			return new FailedTestRetrier(retryingTest.value());
+
+			int maxAttempts = retryingTest.maxAttempts();
+			String maxAttemptsField = "maxAttempts";
```

Comment:
I don't like this. It's `maxAttempts` that you can set through `value` for convenience. Other than that, it's good.

Reply:
Ok, I will change the error to always say "maxAttempts". I was just trying to reference the attribute the user actually used in the annotation.